cmd/tools/releaser/releaser.go

```diff
@@ -0,0 +1,238 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli/v2"
+	"go.uber.org/zap"
+
+	fs2 "github.com/uber/cadence/cmd/tools/releaser/internal/fs"
+	git2 "github.com/uber/cadence/cmd/tools/releaser/internal/git"
```

Comment:
kinda strange to alias these

Reply:
Goland added those when I was refactoring out CLI-wrapped calls. Fixed.